angular-spa-ui/js/search-app/tags-module/tags-cfg.js

```diff
@@ -0,0 +1,12 @@
+var tagsCtrl = require('./tags-ctrl');
+
+var stateProvider = function($stateProvider) {
+    $stateProvider
+        .state('tags', {
+            url: '/tags',
+            templateUrl: '/js/search-app/tags-module/tags.jade',
+            controller: tagsCtrl
+        });
+}
+
+module.exports = stateProvider;
```

Comment:
We decided that module should looks next way:

module.exports = function ([$arg1, ..., $argn]) {
    // Your code here
}